README.md

```diff
@@ -10,6 +10,53 @@ Rhymes with _daily_.
 
 Ailly's best feature is rapid prompt engineering iteration. By keeping your prompts in snippets on the file system, you can make very fine-grained changes to your prompt and immediately see the difference in the output. You can also use all of your normal source control tooling to track changes over time: both your changes, and those from the LLM.
 
+## VSCode extension
+
+The VSCode extension provides several commands.
+
+`Run` runs Ailly on a file. You can activate it with either the right-click context menu or the command palette.
```

Comment:
(Global) The actual commands in the context menu are "Ailly: Run," etc. so I would make sure to match the text of the commands you describe in the README to the actual text in the UI.